test/config.sector.yaml

```diff
@@ -18,7 +18,7 @@ scenario:
   simpl: [""]
   ll: ["copt"]
   clusters: [4]
-  opts: [Co2L-24h]
+  opts: [CCL-Co2L-24h]
```

Comment:
I think it's a great implementation to keep CO2 constraint and the time aggregation wildcard together (since those two are most frequently used often seen in a combination). Could it be a good idea to revise this order also in the docstring of `add_CCL_constraints` for consistency? (this line https://github.com/pypsa-meets-earth/pypsa-earth/blob/4bfb13da0f24a5fc98f541babe00dfd72f7ba3f8/scripts/solve_network.py#L208C16-L208C20)